Makefile

```diff
@@ -259,6 +262,14 @@ bin/golangci-lint: GOLANGCI_LINT_VERSION?=$(shell cat .github/workflows/golangci
 bin/golangci-lint:
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s $(GOLANGCI_LINT_VERSION)
 
+.PHONY: vulncheck
+vulncheck: govulncheck
+	$(TOOLS_BIN_DIR)/govulncheck ./...
+
+govulncheck: ## Download and install govulncheck
+govulncheck:
+	$(call go-get-tool,$(GO_VULNCHECK),golang.org/x/vuln/cmd/govulncheck@latest)
```

Comment:
can we pin the release here? I tried this thing on a 1.17 project and latest was for 1.18 I thought.

Reply:
oh so looking more closely at it, it's still experimental and they're only on v0 without tagged versions. Additionally it only supports Go 1.18 and greater from release. 

https://pkg.go.dev/golang.org/x/vuln/cmd/govulncheck